libs/expo/betterangels/src/lib/screens/Client/Docs/UploadModal/types.ts

```diff
@@ -0,0 +1,42 @@
+import { ReactNativeFile } from '@monorepo/expo/shared/apollo';
+import { ReactNode } from 'react';
+import { ClientProfileQuery } from '../../__generated__/Client.generated';
+
+export interface ISectionProps {
+  onCancel: () => void;
+  title: string;
+  subtitle?: string;
+  onSubmit: () => void;
+  children: ReactNode;
+  loading?: boolean;
+}
+
+export interface IUploadModalProps {
+  isModalVisible: boolean;
+  closeModal: () => void;
+  bottomSection?: React.ReactNode;
+  topSection?: React.ReactNode;
+  opacity?: number;
+  client: ClientProfileQuery | undefined;
+}
+
+export type Docs = {
+  driverLicenseFront: ReactNativeFile | undefined;
+  driverLicenseBack: ReactNativeFile | undefined;
+  birthCertificate: ReactNativeFile | undefined;
+  photoId: ReactNativeFile | undefined;
+  ssn: ReactNativeFile | undefined;
+  consentForms: ReactNativeFile[] | undefined;
+  hmisForms: ReactNativeFile[] | undefined;
+  incomeForms: ReactNativeFile[] | undefined;
+};
+
+export type ITab =
```

Comment:
dl has 2, even if I write driverLicense it will be different because it contains 2 front and back. 
bc was too long I wrote shorten as ssn.

Reply:
updated

Reply:
> dl has 2, even if I write driverLicense it will be different because it contains 2 front and back. bc was too long I wrote shorten as ssn.

should this list have both `DriversLicenseFront` and `DriversLicenseBack`, in that case?

Reply:
> dl has 2, even if I write driverLicense it will be different because it contains 2 front and back. bc was too long I wrote shorten as ssn.

should this list have both `DriversLicenseFront` and `DriversLicenseBack`, in that case?

Reply:
no because that is a tab and not a file